Tests/Functional/FunctionalTests.xml

```diff
@@ -0,0 +1,42 @@
+<phpunit
```

Comment:
AFAIK, the PHPUnit configuration (at least as written by PHPUnit) contains the XML prologue and the schema information:

```xml
<?xml version="1.0" encoding="UTF-8"?>
<phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
         xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/9.6/phpunit.xsd"
```